setup.cfg

```diff
@@ -167,15 +167,6 @@ doctest_subpackage_requires =
     astropy/io/misc/asdf/* = asdf
     astropy/table/mixins/dask.py = dask
 
-[flake8]
```

Comment:
👎 to removing this. My editor actually uses this for "live" PEP 8 checks.

Reply:
Same.

Reply:
kk. I'll leave it in.
FYI, many editors, e.g. VSCode, also support ruff for live PEP8 checks. Ruff is so fast that it has a live mode that can run in real-time.  https://github.com/charliermarsh/ruff#usage